config/webpack.config.js

```diff
@@ -236,9 +241,20 @@ module.exports = env => {
       }),
 
       new MiniCssExtractPlugin({
-        filename: !isOptimizedBuild
-          ? '[name].css'
-          : `[name].[contenthash]-${timestamp}.css`,
+        moduleFilename: chunk => {
+          const { name } = chunk;
+          const standardFilename = `${name}.[contenthash]-${timestamp}.css`;
+          const isMedalliaStyleFile = name === vaMedaliaStylesFilename;
+          const isStandardFile = !isMedalliaStyleFile;
+          const showHash = isOptimizedBuild && isStandardFile;
+
+          if (showHash) return standardFilename;
+
+          const isStaging =
```

Comment:
replaced temp variable with environment helper in 0412326a8c5d81c670da44f33c43c7261e83ff1a

Reply:
I went back to using a temp variable instead of the environment helper function in f832dd3f7bd09395c86b9ea15a1b81f555e7cdb2. 

The [build was failing](https://app.circleci.com/pipelines/github/department-of-veterans-affairs/vets-website/14370/workflows/7637a22e-c24d-480d-b719-c20fdf082156/jobs/110434/parallel-runs/0/steps/0-105). I suspect it's because the helper function uses an `import` statement, and we're using `node@10`, which doesn't support `import` statements. 
```
import ENVIRONMENTS from 'site/constants/environments';
       ^^^^^^^^^^^^
SyntaxError: Unexpected identifier
```

![image](https://user-images.githubusercontent.com/6130520/94846819-60207280-03e7-11eb-952d-0a10dcc27fcc.png)